internal/service/service.go

```diff
@@ -36,14 +40,22 @@ func (s *Service) CreateRetrospective(ctx context.Context, retro *types.Retrospe
 }
 
 func (s *Service) GetRetrospective(ctx context.Context, id uuid.UUID) (*types.Retrospective, error) {
-	return s.repository.GetRetrospective(ctx, id)
+	config := config.Get()
+	cleanUpDays := time.Duration(config.Schedule.CleanUpDays)
+
+	retro, err := s.repository.GetRetrospective(ctx, id)
+	retro.ExpireAt = retro.CreatedAt.Add(cleanUpDays * 24 * time.Hour)
+	return retro, err
 }
 
 func (s *Service) DeleteRetrospective(ctx context.Context, id uuid.UUID) (*types.Retrospective, error) {
+	config := config.Get()
+	cleanUpDays := time.Duration(config.Schedule.CleanUpDays)
 	retro, err := s.repository.DeleteRetrospective(ctx, id)
 	if err != nil {
 		return nil, err
 	}
+	retro.ExpireAt = retro.CreatedAt.Add(cleanUpDays * 24 * time.Hour)
```

Comment:
Why is this here?

Reply:
It can be used to frontend eventually to show when the retro will expire.

As cleanUpDays is a config value, it easier to backend send this.

Reply:
Obs. Technically this information is not necessary in this function, I just insert this because it returns the struct retro.